pmd-cli/src/main/java/net/sourceforge/pmd/cli/PMDCLI.java

```diff
@@ -0,0 +1,37 @@
+package net.sourceforge.pmd.cli;
+
+import net.sourceforge.pmd.cli.commands.internal.PMDRootCommand;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+public class PMDCLI {
```

Comment:
Please use camel case for abbreviations in new classes:
```suggestion
public class PmdCli {
```